boards/arm/google_twinkie_v2/google_twinkie_v2_defconfig

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 The ChromiumOS Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_STM32G0X=y
+CONFIG_SOC_STM32G0B1XX=y
+
+# Serial Drivers
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
```

Comment:
You probably want to enable the USB stack somewhere around here since you have that as default for shell and console, right now the shell sample fails to build.